HIP/hip-540.md

```diff
@@ -94,21 +94,42 @@ First, let's address important language to set a clear distinction between "remo
 ### Other Considerations
 
 1. We should standardize the use of all-zeros key as an invalid key across all keys in Hedera.
-2. We will have to add a boolean flag in the update transaction about whether the system should check the validity of the updated key. Currently, we test the validity for all updates. With this boolean flag, the user will be able to tell the system to avoid checking of the validity of the updated key. The default value of the flag will maintain the current behavior of checking the validity.
+2. We will have to add an enum field in the update transaction about whether the system should check the validity of the updated key. Currently, we test the validity for all updates. With this enum field, the user will be able to tell the system to avoid checking of the validity of the updated key. The default value of the field will maintain the current behavior of checking the validity.
 
 Here's a simple code example illustrating this boolean when updating a key to an invalid key.
 
 ```js
 const newSupplyKey = "0x0000000000000000000000000000000000000000";
 let tokenUpdateTx = await new TokenUpdateTransaction()
     .setTokenId(tokenId)
-    // Interface (key: Key, shouldVerifyKey: boolean = true) 
-    // When set to false, the TokenUpdateTransaction won't check for the validity of the key
-    .setSupplyKey(newSupplyKey, false)
+    .setSupplyKey(newSupplyKey)
+    // Interface (shouldVerifyKey: enum = FULL_VALIDATION)
+    // When set to NO_VALIDATION, the TokenUpdateTransaction won't check for the validity of the key
+    .shouldVerifyKey(NO_VALIDATION)
```

Comment:
If we rename the property, this would become `.setKeyVerificationMode(NO_VALIDATION)`